recipe/README.md

```diff
@@ -0,0 +1,83 @@
+Building the conda package
+==========================
+This directory contains the configuration files that are needed to build
+orca into a standalone conda package.
+
+To build the conda package, first install
+[Anaconda](https://www.anaconda.com/download/#macos) or
+[Miniconda](https://conda.io/miniconda.html). 
+
+Next, use `conda` to install the `conda-build` package:
+
+```bash
+$ conda install conda-build
+```
+
+Finally, build the package from the root project directory:
+
+```bash
+$ conda build recipe/
+```
+
+The resulting package will be named `plotly-orca-*.tar.bz2`, and the build
+command will display the full path of the output location. 
+
+How it works
```

Comment:
Wow. I didn't expect this much detail. Thanks for your efforts!

Reply:
haha, sure. Once I got started a bunch of details came to mind that I thought would be useful background for future contributors.